curious/dataclasses/embed.py

```diff
@@ -151,8 +151,8 @@ def set_image(self, *, image_url: str) -> 'Embed':
         """
         self.image = attrdict()
 
-        if not image_url.startswith("http") or image_url.startswith("attachment://"):
-            raise ValueError("Image URLs must start with http[s]")
+        if not (image_url.startswith("http") or image_url.startswith("attachment://")):
```

Comment:
What's with the extra brackets?

Reply:
`not False or True` is `True`, we want `not (False or True)`, which is `False`

Reply:
Oh okay I don't know how to program